fdbclient/AuditUtils.actor.cpp

```diff
@@ -458,6 +490,14 @@ ACTOR Future<Void> persistAuditStateByRange(Database cx, AuditStorageState audit
 		try {
 			tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
 			tr.setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
+			Optional<Value> res_ = wait(tr.get(auditKey(auditState.getType(), auditState.id)));
+			ASSERT(res_.present());
```

Comment:
In general we don't want to check on `data`, can we make this test-only, and handle the missing audit key more gracefully? E.g., throw `audit_storage_failed`

Reply:
Agree.